src/ape/api/accounts.py

```diff
@@ -93,10 +93,16 @@ def call(self, txn: TransactionAPI, send_everything: bool = False) -> ReceiptAPI
         txn = self.prepare_transaction(txn)
 
         if send_everything:
+            error_message = "Sender does not have enough to cover transaction value and gas: "
```

Comment:
Is this used anywhere other than [here](https://github.com/ApeWorX/ape/pull/731/files#diff-c308960cdf9376a4c05b2bb028a5a79e22c8a12b4c99633580062ec04ab613e2R104)?

Reply:
no, I can refactor this

Reply:
refactored